### PR TITLE
add tosdr message to popup

### PR DESCRIPTION
**Pull request by jdorweiler**
**Reviewer:**

## Description:

Add tosdr message to popup grade explanation
screen shot 2017-11-08 at 6 19 04 pm

## Steps to test this PR:

Some test cases

| site | tosdr.class | message |
|----------------|-------------|-------------|
| duckduckgo.com | A           | Good        |
| github.com     | B           | Mixed       |
| google.com     | C           | Poor        |
| youtube.com     | D          | Poor        |
| reddit.com     | -           | Unknown     |

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: js/ui/models/site.es6.js [hunk 1]
```diff
@@ -45,6 +46,7 @@ Site.prototype = $.extend({},
                           }
                           this.setSiteProperties()
                           this.setHttpsMessage()
+                          this.tosdr = backgroundTabObj.site.score.tosdr
```
Comment (laurengarcia) [inline js/ui/models/site.es6.js]: This needs to go above here, inside of the guard `if (backgroundTabObj) ...`

File: js/ui/templates/grade-details.es6.js [hunk 2]
```diff
@@ -19,6 +20,14 @@ module.exports = function () {
                 ${httpsMsg(this.model.site.httpsState)}
                 <div class="float-right"></div>
             </h2>
+            <h2 class="site-info__tosdr-status">
+                ${tosdrMsg(this.model.site.tosdr)}
+                <div class="float-right"></div>
```
Comment (laurengarcia) [inline js/ui/templates/grade-details.es6.js]: What's happening with this empty div here?

File: js/ui/templates/grade-details.es6.js [hunk 3]
```diff
@@ -19,6 +20,14 @@ module.exports = function () {
...
+            </h2>
+            <p class="site-info__tosdr-msg padded border--bottom">
+                Using privacy policy analysis from TOSDR
+                <div class="float-right"></div>
```
Comment (laurengarcia) [inline js/ui/templates/grade-details.es6.js]: What's happening with this empty div here?

File: js/ui/models/site.es6.js (hunk 1 above)
Comment (jdorweiler) [reply js/ui/models/site.es6.js]: Yes, thanks.

File: js/ui/templates/grade-details.es6.js (hunk 2 above)
Comment (jdorweiler) [reply js/ui/templates/grade-details.es6.js]: Doesn't seem to do anything but I saw the https message above has one.

**Pull request merged**